src/pkg/whatsapp/whatsapp.go

```diff
@@ -400,7 +413,22 @@ func forwardToWebhook(evt *events.Message) error {
 	if err != nil {
 		return pkgError.WebhookError(fmt.Sprintf("error when create http object %v", err))
 	}
+
+	var secretKey []byte
+	if config.WhatsappWebhookSecret != "" {
+		secretKey = []byte(config.WhatsappWebhookSecret)
+	} else {
+		secretKey = []byte("anything")
+	}
+
+	signature, err := getMessageDigestOrSignature(postBody, secretKey)
+	if err != nil {
+		return pkgError.WebhookError(fmt.Sprintf("error when create signature %v", err))
+	}
+
 	req.Header.Set("Content-Type", "application/json")
+	req.Header.Set("X-Hub-Signature-256", fmt.Sprintf("sha256=%s", signature))
```

Comment:
change name `X-Signature-256` because this is not github webhook

Reply:
`X-Hub-Signature-256` is based on the standard used by Facebook for Whatsapp webhooks. I wanted to keep consistency with their implementation for signature verification.

Let me know if you'd still prefer the header to be changed to `X-Signature-256`. I'm happy to adjust it.

Ref: https://developers.facebook.com/docs/messenger-platform/webhooks/#validate-payloads